components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -115,6 +116,10 @@ const BillingSubscriptions = () => {
         if (actionQueryParam === 'show_pricing_modal') {
             openPricingModal({trackingLocation: 'billing_subscriptions_external_direct_link'});
         }
+
+        if (actionQueryParam === 'show_delinquency_modal') {
+            openCloudPurchaseModal({trackingLocation: 'billing_subscriptions_external_direct_link'});
+        }
```

Comment:
Did we update the url in the emails to include this new param?

Reply:
All but one are already set to this in master, the one that isn't, will be updated here: https://github.com/mattermost/mattermost-server/pull/20925